spotify/models/user.py

```diff
@@ -182,22 +151,16 @@ def _raise(*args, **kwargs):
         raw = await client.http.request(route, headers=headers, params=payload)
 
         token = raw["access_token"]
+        refresh_token = raw["refresh_token"]
 
-        refresh_: Optional[Tuple[int, str]]
-        if refresh:
-            refresh_ = (raw["expires_in"], raw["refresh_token"])
-        else:
-            refresh_ = None
-
-        return await cls.from_token(client, token, refresh=refresh_)
+        return await cls.from_token(client, token, refresh_token)
 
     @classmethod
     async def from_token(
         cls,
         client: "spotify.Client",
-        token: str,
-        *,
-        refresh: Optional[Tuple[int, str]] = None,
+        token: str = None,
```

Comment:
Should this annotation be token: Optional[str]?  The Optional part seems redundant to me since I'm providing default value of None.

Reply:
`Optional[str]` is equivalent to `Union[str, None]`, so yes it should be `Optional[str]`

Reply:
Alright I changed it to Optional[str].  The Optional seems kind of redundant to me since I'm defaulting the value to None.

Reply:
note: https://docs.python.org/3/library/typing.html#typing.Optional